src/compas_fab/sensors/baumer.py

```diff
@@ -433,13 +433,7 @@ def format_command(self, address, command, data=None):
         return self.FRAME_TAIL % (frame, self.calculate_checksum(frame))
 
     def calculate_checksum(self, command):
-        """Checks that message is complete.
-
-        Note
-        ----
-        See `RS-485 Protocol Structure <https://www.baumer.com/ae/en/product-overview/2d-3d-sensors/profile-sensors-/measurement-of-round-objects/\
```

Comment:
Running `check-links` showed this one as 404, and I couldn't be bother to find it again. 😛

Reply:
i've never looked at this file, but from this snippet, this particular method looks somewhat misleading....

Reply:
The sensor supports checksums on the messages, but it also allows `'****'` to skip checking it.